src/blueprints/tv2/action-factories/tv2-replay-action-factory.ts

```diff
@@ -69,12 +69,12 @@ export class Tv2ReplayActionFactory {
     }
   }
 
-  private removeAllWhitespace(value: string): string {
-    return value.replaceAll(' ', '')
+  private sanitizeString(value: string): string {
+    return value.replaceAll(' ', '').replaceAll('/', '_')
   }
```

Comment:
From `sanitizeString` I don't know what to expect. A suggestion for an alternative name is `sanitizeIdentifier` or `sanitizeSourceName`.

Reply:
I didn't chose your naming suggestions because there is nothing in the body of the function that indicates that it has anything to do with either an `Identifier` or a `SourceName`. So including those in the name would be naming the method after _how we intend to use it_ (which is outside the scope of the method) and not _what does the method do_.

I agree, the name could be more specific. I will try and come up with something.

Reply:
I have made an update :)

Reply:
I have changed the parameter from `string` to `Tv2SourceMappingWithSound`.
This allows us to call it `getSanitizedName()`.